src/tests/bounding_box_test_input.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
to make it more self-contained, please put a comment at the beginning of the file and say what each element of position, rotate_angle and dimension lists is.

Reply:
can we comments in json file? :\

Reply:
apparently not. For now, just add the key "comment" to the first testcase and add the text to it. let's see if we find a better solution later
@hfanahita @erfanz